templates/openstack/features/index.html

```diff
@@ -0,0 +1,541 @@
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+
+  {% call(slot) vf_hero(
+    title_text='Enterprise cloud solution',
+    subtitle_text='Power mission-critical workloads with Canonical OpenStack',
+    layout='50/50-full-width-image'
+  ) -%}
+    {%- if slot == 'description' -%}
+      <p>
+        Gift your enterprise a robust, highly scalable, and flexible cloud solution. Boost the efficiency of your sophisticated workloads, including AI and data, while benefitting from long-term stability and security.
+      </p>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <a href="https://assets.ubuntu.com/v1/cb9568ea-Datasheet%20-%20Canonical%20OpenStack.pdf" class="p-button--positive">Get the datasheet</a>
+    {%- endif -%}
+    {%- if slot == 'image' -%}
+      <div class="p-image-container--cinematic is-cover">
+        <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/30bdbfba-hero.jpg" alt="">
+      </div>
+    {% endif -%}
+  {% endcall -%}
+
+  {{ vf_basic_section(
+    title={"text": "At the forefront of innovation"},
+    is_split_on_medium=True,
+    items=[
+      {
+        "type": "image",
+        "item": {
+          "aspect_ratio": "3-2",
+          "attrs": {
+            "src": "https://assets.ubuntu.com/v1/7eb12d1c-Image%20container.png",
+            "alt": ""
+          }
+        }
+      },
+      {
+        "type": "description",
+        "item": {
+          "type": "html",
+          "content": "
+            <p>
+              When you are experimenting with AI agents, rolling out 5G networks, or looking for a way to protect your data with confidential computing, you need an enterprise cloud solution. We recognize that. This is why Canonical OpenStack is always at the forefront of innovation.
+            </p>
+          "
+        }
+      },
+      {
+        "type": "list",
+        "item": {
+          "list_items": [
+            {
+              "list_item_type": "bullet",
+              "content": "All the goodness of the upstream code packaged into one human-friendly product"
+            },
+            {
+              "list_item_type": "bullet",
+              "content": "Get an OpenStack that has been pushed to its limits across various market sectors and the most sophisticated use cases"
+            },
+            {
+              "list_item_type": "bullet",
+              "content": "With us, your cloud is backed by the third biggest contributor ever and developed in cooperation with our strategic partners"
+            }
+          ]
+        }
+      },
+      {
+        "type": "cta-block",
+        "item": {
+          "link": {
+            "content_html": "Discover Canonical OpenStack&nbsp;&rsaquo;",
+            "attrs": {
+              "href": "/openstack"
+            }
+          }
+        }
+      }
+    ]
+  ) }}
+
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row">
+        <hr class="p-rule" />
```

Comment:
```suggestion
        <hr class="p-rule--muted" />
```
The secondary rules here seem to be muted